docs/tutorials/single-dev-workflow.md

```diff
@@ -0,0 +1,206 @@
+
+As [explained here](../explanations/detailed-push-process.md#7-stage), Epinio uses [Paketo buildpacks](https://paketo.io/) to create a container image for your application. This image is then used to create a container with your application, which will run on your local Kubernetes cluster.
+
+Epinio will also create a new `ingress route`, which will allow you to easily access your application once it's deployed.
+
+The whole process is handled by Epinio, which enables you to concentrate on your application rather than knowing how you'll be able to deploy it.
+
+Let's see first how to deploy a simple application:
+
+```shell
+# Example code: https://github.com/epinio/example-12factor
+
+# Move to the source code directory. Here is an example:
+cd /github/example-12factor
+
+# Deploy your application
+epinio push -n mysimpleapp
+```
+
+At the end of the deployment output, you have the URL to be used for checking your application:
+
+```shell
+Deploying application ...
+..............
+🕞  Creating application resources
+✔️  App is online.
+Name: mysimpleapp
+Namespace: workspace
+Builder Image:
+Routes:
+1: https://mysimpleapp.127.0.0.1.sslip.io
+```
+
+### List the applications deployed
+
+If you're working on many applications, it can be really useful to see when was the last time they were deployed and which URL you should use to check them.
+
+You can get the application's information with the following two commands:
+
+```shell
+# List all deployed applications
+epinio app list
+
+# Display the applications information
+epinio app show mysimpleapp
+```
+
+### View installation logs
+
+If your application couldn't be installed, you might want to check your installation logs or, even better, save them into a file for a better screening with text editor.
+
+You can access the logs by running the command:
+
+```shell
+# Get the installation logs
+epinio app logs --staging mysimpleapp
+
+# Get the application logs
+epinio app logs --follow mysimpleapp
+```
+
+### View application logs
+
+Another type of logs that you can access is the application logs. And specially with web applications, you might want to have realtime logs displaying so you spot bugs faster.
+
+Epinio can display the logs both statically or dynamically as follows:
+
+```shell
+# Display logs statically
+epinio app logs mysimpleapp
+
+# Display logs dynamically
+epinio app logs --follow mysimpleapp
+```
+
+### Create a new port-forward
+
+As described above, Epinio creates a new `ingress route` for your application. The route is bound by default to the port `443`.
+
+However, you might need to test parts of your application using a different port. For these specific cases, you can run the following command:
+
+```shell
+epinio app port-forward 8080:8080 mysimpleapp
+```
+
+:::tip
+
+You can specify only one port number. In that case, Epinio will open the port of both `local` and `remote` targets.
```

Comment:
It might be worth noting that by default all applications are listening inside of their pod on port 8080 so specifying both is useful for when you are forwarding multiple apps at the same time.

Reply:
I do agree, however as port forwarding will throw an error if the port is already used on the host, then I think it's a bit outside the scope of this tutorial.